builder/parser/line_parsers.go

```diff
@@ -149,7 +149,7 @@ func parseEnv(rest string) (*Node, map[string]bool, error) {
 		strs := TOKEN_WHITESPACE.Split(rest, 2)
 
 		if len(strs) < 2 {
-			return nil, nil, fmt.Errorf("ENV must have two arguments")
+			return nil, nil, fmt.Errorf(key + " must have two arguments")
```

Comment:
Don't know what to say because it does just build of off the env logic.  Does env have the same oddity or is there something specific to labels that added this.  Frankly I think `--label foo` is nice and wouldn't want to remove that.  So I'd prefer the Dockerfile support the same, but then I couldn't reuse the env parser.